lib/EventListener/ObjectUpdatedEventListener.php

```diff
@@ -25,10 +24,29 @@ public function __construct(
     public function handle(Event $event): void
     {
         if ($event instanceof ObjectUpdatedEvent === false) {
-			return;
-		}
+            return;
+        }
 
-		$object = $event->getNewObject();
-		$this->synchronizationService->synchronizeToTarget($object);
+        if (method_exists($event, 'getObject') === false) {
+            return;
+        }
+
+
+        $object = $event->getObject();
+        if ($object === null || $object->getRegister() === null || $object->getSchema() === null && $object->getObject() !== null) {
+            return;
+        }
+
+        $synchronizations = $this->synchronizationService->findAllBySourceId(register: $object->getRegister(), schema: $object->getSchema());
+        foreach ($synchronizations as $synchronization) {
+            try {
+                $this->synchronizationService->synchronize($synchronization, false, true, $object);
```

Comment:
Could you add variable names here? That makes the code better readable.

Reply:
Better now. Not necessary at this time: but you know this way you can leave out variables that have to have their default value?